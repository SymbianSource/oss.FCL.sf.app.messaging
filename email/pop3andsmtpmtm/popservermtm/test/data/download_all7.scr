+OK demon POP3 server ready (demon.co.uk)
+OK 
+OK 
+OK 38 872441
+OK 38 messages
1 2348
2 1680
3 1134
4 3765
5 4583
6 1613
7 1710
8 2377
9 2376
10 1611
11 1613
12 2378
13 2376
14 1708
15 1608
16 2376
17 4579
18 1613
19 1709
20 2377
21 1612
22 1611
23 2377
24 1581
25 417291
26 1424
27 2926
28 1297
29 1003
30 1705
31 1201
32 1709
33 2372
34 1612
35 1708
36 1706
37 1709
38 382063
.
+OK 38 messages
1 psion7.951245177:10:21094:553
2 psion7.951245179:10:21094:555
3 psion7.951310224:10:27493:17
4 psion7.951310332:10:28042:25
5 psion7.951310765:10:16898:10
6 psion7.951227817:11:26824:168
7 psion7.951226726:11:17469:185
8 psion7.951225887:10:13079:27
9 psion7.951229477:10:10412:0
10 psion7.951227361:11:07708:162
11 psion7.951227092:10:15520:175
12 psion7.951227031:10:11352:207
13 psion7.951233922:11:19305:338
14 psion7.951232724:11:02400:500
15 psion7.951227783:10:08245:29
16 psion7.951229007:11:29843:185
17 psion7.951236742:10:00158:371
18 psion7.951229716:11:05903:515
19 psion7.951231100:11:27837:98
20 psion7.951226457:11:17507:44
21 psion7.951227768:11:04075:47
22 psion7.951235204:11:21705:825
23 psion7.951226862:10:08453:56
24 psion7.951242802:20:05688:1
25 psion7.951305385:20:09162:36
26 psion7.951310087:20:27915:20
27 psion7.951310089:20:27208:74
28 psion7.951310090:20:27898:25
29 psion7.951310095:20:27897:19
30 psion7.951226989:20:00379:48
31 psion7.951233515:20:07705:2
32 psion7.951225769:20:15803:20
33 psion7.951226992:20:00912:1
34 psion7.951225775:20:14982:63
35 psion7.951226429:20:11339:2
36 psion7.951227233:20:06668:9
37 psion7.951226664:20:18615:63
38 psion7.951554999:20:07849:2
.
-ERR Unknown Command "CAPA"
-Err (Can't get the TOP of the first mail)
