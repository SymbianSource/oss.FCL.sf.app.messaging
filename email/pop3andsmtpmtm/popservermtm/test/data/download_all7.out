STAT
LIST
UIDL
CAPA
TOP 38 1
QUIT
